src/js/edu-benefits/components/veteran-information/VeteranInformationFields.jsx

```diff
@@ -24,9 +26,12 @@ export default class PersonalInformationFields extends React.Component {
               month={this.props.data.veteranDateOfBirth.month}
               year={this.props.data.veteranDateOfBirth.year}
               onValueChange={(update) => {this.props.onStateChange('veteranDateOfBirth', update);}}/>
-          <Gender
+          <ErrorableRadioButtons
```

Comment:
@akainic I changed Gender to this because @cweber and @goldenmeanie wanted the gender field to use radio buttons. I could also just change the Gender component. Thoughts?